### PR TITLE
Added description of XAL to the front page.

### PR DESCRIPTION
**Pull request by kememil**

### CODE REVIEW

**Comment by kememil** [comment]
This PR is related to PR #122 where the description of XAL has been updated. The old text from the About us section has been moved to the front page.

File: content/_index.md [hunk 1]
```diff
@@ -21,10 +21,9 @@ boxes:
...
-  Get in touch with us either by e-mail to
-  [post@expertanalytics.no](mailto:post@expertanalytics.no)
-  or by calling us at +47 900 69 915.
-{{< /hide-if-mobile >}}
+Expert Analytics is a high tech software consultancy based in Oslo, Norway. We offer
+quality software solutions for industrial applications to both domestic and
+international customers. Our experts have years of hands-on programming experience
+combined with a solid background in the natural sciences, which translates into an
+ability to analyse and solve complex challenges. For the client, it guarantees
+that our solutions will meet their needs and beyond.
```
Comment (skavhaug) [inline content/_index.md:29]: I do not really like this text. What do we do to improve?

**Review by skavhaug — CHANGES_REQUESTED** [review]

File: content/_index.md (hunk 1 above)
Comment (kememil) [reply content/_index.md:29]: I can agree to that. This is the old text from the About us section and we should decide on what to put on the front page and what to leave for the About section. See #122 for the updated text in the About us section.

Comment (skavhaug) [reply content/_index.md:29]: Yeah, now that we actually go though this and improve, we should take the opportunity to freshen it up.

Comment (kememil) [reply content/_index.md:29]: How about this one from the slide deck for the front page:
On demand Expertise in Scientific & Industrial Data Analytics

Comment (skavhaug) [reply content/_index.md:29]: That's a good start. Please feel free to try out some formulations, and I'll do my best to help out :)

File: content/_index.md [hunk 2]
```diff
@@ -21,10 +21,14 @@ boxes:
     icon: assets/memory.svg
 ---
 
-Møllergata 8, 0179, Oslo, Norway
+On Demand Expertise in Scientific & Industrial Data Analytics
```
Comment (eigilhs) [inline content/_index.md:24]: I think "on-demand" should be hyphenated when used as an adjective and placed before the noun.
```suggestion
On-Demand Expertise in Scientific & Industrial Data Analytics
```

**Review by mandus — APPROVED** [review]
I'm happy with this - and it's better than what we have, so let's go with it?

File: content/_index.md (hunk 1 above)
Comment (skavhaug) [reply content/_index.md]: Let's remove the last sentence, and I'm happy to approve this PR ;)

**Review by skavhaug — CHANGES_REQUESTED** [review]

**Review by skavhaug — APPROVED** [review]

**Pull request merged**